Dockerfile

```diff
@@ -41,8 +43,7 @@ pkg-config \
 protobuf-compiler-grpc \
 libgrpc++1.51 \
 libgrpc++-dev \
-libpcap0.8-dev \
-linux-headers-${OSARCH}
```

Comment:
I could not test in live environment, so do not know what this will change. I guess nobody knows anymore why this was here?

Reply:
This was never included before, as the OSARCH was not set in old builds without the changes in this PR.
That is why I removed it.
If you take a look at the older builds without this change, you can see that it was never installed:
https://github.com/ironcore-dev/dpservice/actions/runs/7744421590/job/21118124520
![image](https://github.com/ironcore-dev/dpservice/assets/1742013/bebff2ce-1fff-41ec-82b7-f49283c217c1)
![image](https://github.com/ironcore-dev/dpservice/assets/1742013/6a71c719-3947-4e48-a1ca-01df570b1783)